devices.js

```diff
@@ -9631,6 +9631,11 @@ const devices = [
         exposes: [e.action(['on', 'off', 'brightness_step_up', 'brightness_step_down', 'brightness_move_up', 'brightness_move_down',
             'brightness_stop'])],
         toZigbee: [],
+        meta: {configureKey: 1},
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic', 'genOnOff', 'genLevelCtrl', 'genScenes']);
```

Comment:
Can you also try adding `genPowerCfg` here and `fz.battery` to the `fromZigbee` list above? Do you get battery reports within 24 hours?

Reply:
If "configureKey" is set to 2 this error periodically occurs (independent of adding genPowerCfg and fz.battery): 
`Zigbee2MQTT:error 2021-01-06 20:37:24: Failed to configure '0xSERIAL', attempt 2 (Error: Bind 0xSERIAL/1 genBasic from '0xSERIAL/1' failed (AREQ - ZDO - bindRsp after 10000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/dist/utils/waitress.js:46:35)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7))`

I did not yet test adding "await reporting.batteryPercentageRemaining(endpoint);"

Reply:
`    {
        zigbeeModel: ['ZBT-DIMController-D0800'],
        model: '404002',
        description: 'Tint dim remote control',
        vendor: 'Müller Licht',
        fromZigbee: [fz.command_on, fz.command_off, fz.command_step, fz.command_move, fz.command_stop, fz.battery],
        exposes: [e.action(['on', 'off', 'brightness_step_up', 'brightness_step_down', 'brightness_move_up', 'brightness_move_down',
            'brightness_stop'])],
        toZigbee: [],
        meta: {configureKey: 2},
        configure: async (device, coordinatorEndpoint, logger) => {
            const endpoint = device.getEndpoint(1);
            await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic', 'genOnOff', 'genLevelCtrl', 'genScenes', 'genPowerCfg']);
            await reporting.batteryPercentageRemaining(endpoint);
        },
    },`

I get following error periodically:

`Zigbee2MQTT:error 2021-01-06 20:45:22: Failed to configure '0xSERIAL', attempt 1 (Error: Bind 0xSERIAL/1 genPowerCfg from '0xSERIAL/1' failed (AREQ - ZDO - bindRsp after 10000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/dist/utils/waitress.js:46:35)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7))
`

Reply:
Can you provide the data/database.db entry of this device?

Reply:
```javascript
{"id":2,"type":"EndDevice","ieeeAddr":"0xSERIAL","nwkAddr":42771,"manufId":4456,"manufName":"LDS","powerSource":"Battery","modelId":"ZBT-DIMController-D0800","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":2096,"inClusterList":[0,4096,3],"outClusterList":[3,4,5,6,8,4096,25],"clusters":{"genBasic":{"attributes":{"modelId":"ZBT-DIMController-D0800","manufacturerName":"LDS","powerSource":3,"zclVersion":2,"appVersion":1,"stackVersion":1,"hwVersion":1,"dateCode":"20180611-5","swBuildId":"2.0"}}},"binds":[{"cluster":0,"type":"endpoint","deviceIeeeAddress":"0xADDR","endpointID":1},{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0xADDR","endpointID":1},{"cluster":8,"type":"endpoint","deviceIeeeAddress":"0xADDR","endpointID":1},{"cluster":5,"type":"endpoint","deviceIeeeAddress":"0xADDR","endpointID":1}],"configuredReportings":[],"meta":{}}},"appVersion":1,"stackVersion":1,"hwVersion":1,"dateCode":"20180611-5","swBuildId":"2.0","zclVersion":2,"interviewCompleted":true,"meta":{"configured":1},"lastSeen":1609964929170}
```

I just blanked the specific serial numbers but I can provide them if you need them.

Reply:
Thanks, it doesn't support battery measurements indeed as the `genPowerCfg` (ID: 1) is not in the `outClustersList`.

Reply:
Ah ok, thank you for your efforts! Unfortunately, I don't know where to look such things up (I mainly looked at other examples in the devices.js and the description for new devices in the documentation). Usually, this is not required as most of the time zigbee2mqtt works very well ;) .